packages/platform-browser/animations/async/src/async_animation_renderer.ts

```diff
@@ -35,8 +38,11 @@ export class AsyncAnimationRendererFactory implements RendererFactory2 {
 
     return moduleImpl
         .catch((e) => {
-          // TODO: Create a runtime error
-          throw new Error('Failed to load the @angular/animations/browser module');
+          throw new RuntimeError(
+              RuntimeErrorCode.ANIMATION_RENDERER_ASYNC_LOADING_FAILURE,
+              'You are using the `provideAnimationsAsync()` provider. ' +
+                  'Angular failed to load the animation renderer. ' +
+                  'No animations will be displayed and their style won\'t be applied.');
```

Comment:
It make sense to keep the error in prod bundles, but we can tree-shake-away the error message:
```suggestion
              (typeof ngDevMode === 'undefined' || ngDevMode) && 'Async loading for animations package was ' + 
                  'enabled, but loading failed. Angular falls back to using regular rendering. ' +
                  'No animations will be displayed and their styles won\'t be applied.');
```
(I've also rephrased the error message a bit)

If we can safely fall back to a regular rendering, I'm wondering if this `throw new RuntimeError` should be a `console.error` instead? WDYT? We should not change it in this PR though, since this is unrelated to the change.

Reply:
Yes, I'll convert it to a standard `console.error` in an upcomming PR.